spec/factories/file_type.rb

```diff
@@ -124,6 +124,16 @@
     singleton_file_type
   end
 
+  factory :dot_pylintrc, class: 'FileType' do
+    created_by_admin
+    editor_mode { 'ace/mode/plain_text' }
+    executable
```

Comment:
Why is the `executable` flag necessary here?

Reply:
removed